Dockerfile

```diff
@@ -0,0 +1,38 @@
+#Build Stage
```

Comment:
Please don't use git clone or `ldd`. You don't need to set the AFL env vars at the end here.